azext_edge/edge/params.py

```diff
@@ -506,11 +512,15 @@ def load_iotops_arguments(self, _):
         context.argument(
             "session_expiry",
             options_list=["--session-expiry"],
-            help="The session expiry interval in seconds for the data flow MQTT client. " "Min value: 0.",
+            help="The session expiry interval in seconds for the data flow MQTT client. "
+            "Min value: 0.",
```

Comment:
I know the split string is from some auto formatting issue, we should combine the strings.